Avoid the optional digest feature for now.

We just want to build the binary sha1cdsum and ship the standard
sha1collisiondetection crate, and shipping the digest-trait feature
would require an upgrade of a large number of rust-crypto packages.
Once those updates are done, then we can add the digest-trait feature
back.

Also, the structopt "feature" is really just a build-dependency for
the sha1cdsum binary, so we don't want it as an additional feature.

debcargo doesn't seem to have a way to represent this cleanly
(#977491) so we'll just treat it as non-optional for now.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,22 +29,14 @@
 [[bin]]
 name = "sha1cdsum"
 path = "src/main.rs"
-required-features = ["std", "structopt"]
-[dependencies.digest]
-version = "0.9"
-optional = true
+required-features = ["std"]
 
 [dependencies.generic-array]
 version = ">= 0.12, < 0.15"
 
-[dependencies.sha-1]
-version = "0.9"
-optional = true
-
 [dependencies.structopt]
 version = "0.3"
-optional = true
-default-features = false
+
 [dev-dependencies.getrandom]
 version = "0.2"
 
@@ -52,9 +44,9 @@
 version = "0.3"
 
 [features]
-default = ["std", "digest-trait", "structopt"]
-digest-trait = ["digest", "sha-1"]
-std = ["digest/std"]
+default = ["std"]
+std = []
+
 [badges.gitlab]
 repository = "sequoia-pgp/sha1collisiondetection"
 
